cmd/migrator/app/migrator.go

```diff
@@ -24,8 +23,8 @@ const (
 
 var (
 	kubeconfigPath = flag.String("kubeconfig", "", "absolute path to the kubeconfig file specifying the apiserver instance. If unspecified, fallback to in-cluster configuration")
-	kubeAPIQPS     = flag.Float32("kube-api-qps", rest.DefaultQPS, "QPS to use while talking with kubernetes apiserver.")
-	kubeAPIBurst   = flag.Int("kube-api-burst", rest.DefaultBurst, "Burst to use while talking with kubernetes apiserver.")
+	kubeAPIQPS     = flag.Float32("kube-api-qps", 40.0, "QPS to use while talking with kubernetes apiserver.")
```

Comment:
@sanchezl get this fixed upstream

Reply:
@deads2k I was planning on adding the `--kube-api-qps` and `"kube-api-burst` options on our invocation of the migrator and then just skipping this patch on the next rebase.